docs/cue/reference/services/prometheus_remote_write.cue

```diff
@@ -0,0 +1,16 @@
+package metadata
+
+services: prometheus_remote_write: {
+	name:     "Prometheus Remote Write Integration"
+	thing:    "a metrics database or service"
+	url:      urls.prometheus_remote_integrations
+	versions: null
+
+	description: """
+		Databases and services that are capable of receiving data via the Prometheus
+		[`remote_write protocol`](\(urls.prometheus_remote_write_protocol)).
+
+		Prometheus itself can also receive metrics via the `remote_write` protocol if the feature is enabled via
```

Comment:
For clarification, does this feature need to be enabled at compile time (ie `configure --enable…`) or at run time?

Reply:
It's a runtime flag:

```
docker run --rm -it prom/prometheus --enable-feature=remote-write-receiver
...
level=info ts=2021-07-16T12:47:42.623Z caller=main.go:149 msg="Experimental remote-write-receiver enabled"
```